components/post_view/channel_intro_message/add_members_call_to_action.tsx

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import {FormattedMessage} from 'react-intl';
+
+import {Channel} from 'mattermost-redux/types/channels';
+
+import {Permissions} from 'mattermost-redux/constants';
+
+import ToggleModalButtonRedux from 'components/toggle_modal_button_redux';
+import ToggleModalButton from 'components/toggle_modal_button.jsx';
+import InvitationModal from 'components/invitation_modal';
+import ChannelInviteModal from 'components/channel_invite_modal';
+import AddGroupsToChannelModal from 'components/add_groups_to_channel_modal';
+import ChannelPermissionGate from 'components/permissions_gates/channel_permission_gate';
+
+import {Constants, ModalIdentifiers} from 'utils/constants';
+import * as Utils from 'utils/utils.jsx';
+
+import './add_members_call_to_action.scss';
+
+import MembersSvg from './members_illustration.svg';
+
+export interface AddMembersCallToActionProps {
+    totalUsers: number;
+    usersLimit: number;
+    channel: Channel;
+    setHeader: React.ReactNode;
+}
+
+const AddMembersCallToAction: React.FC<AddMembersCallToActionProps> = ({totalUsers, usersLimit, channel, setHeader}: AddMembersCallToActionProps) => {
```

Comment:
CTA is more generic - product level wording, maybe something like:

```suggestion
const AddMembersButton: React.FC<AddMembersCallToActionProps> = ({totalUsers, usersLimit, channel, setHeader}: AddMembersCallToActionProps) => {
```